packages/shared/src/components/widgets/SocialShareList.tsx

```diff
@@ -42,7 +35,25 @@ export function SocialShareList({
   onCopy,
   onNativeShare,
   onClickSocial,
```

Comment:
So we don't use this anymore? can we remove?

Reply:
We use it on line 43 now.